### ISSUE TITLE
Resizing a PuTTY window while AFL++ is running causes AFL++ to abort

### ISSUE THREAD

**Issue by geeknik** [AFLplusplus/AFLplusplus#402]
I'm fuzzing with AFL++ built from `12bdef` on a remote VPS and I'm connected using PuTTY v0.73 from a Fedora 31 host. My setting for the PuTTY window is 80 x 26 with a font size of 14. Makes for a pretty comfortable read for my old eyes. I'm also using screen on the remote VPS if that factors into this at all.  (EDIT: It doesn't, AFL++ aborts the same way when run under tmux)

Anyways, I accidentally grabbed the edge of the PuTTY window, causing it to be resized. AFL++ aborted with the following messages:

```
[-] Unable to communicate with fork server. Some possible reasons:

    - You've run out of memory. Use -m to increase the the memory limit
      to something higher than 0.
    - The binary or one of the libraries it uses manages to create
      threads before the forkserver initializes.
    - The binary, at least in some circumstances, exits in a way that
      also kills the parent process - raise() could be the culprit.
    - If using persistent mode with QEMU, AFL_QEMU_PERSISTENT_ADDR is
      probably not valid (hint: add the base address in case of PIE)

If all else fails you can disable the fork server via AFL_NO_FORKSRV=1.

[-] PROGRAM ABORT : Unable to communicate with fork server
         Location : afl_fsrv_run_target(), src/afl-forkserver.c:962
```

I assume this is the result of some weird control code being sent by PuTTY when the window is resized, unsure if there is any way to ignore those?

Now here's where it gets surreal: If I open a local console on the Fedora 31 host and `ssh user@ip` to the same VPS, typing `screen -x fuzz` and restarting AFL++, I can switch back to the PuTTY window and resize the window to span multiple monitors and AFL++ keeps on running and never aborts. Upon switching back over to the local console, tapping `ctrl-a-d` to put screen into the background and then logging out, resizing the PuTTY window again triggers AFL++ to abort with the above message.

**Comment by vanhauser-thc** [comment]
actually the same happened to me today when I resized a local window where afl-fuzz was running.
My guess is it is a resize signal that is not handled.
thanks for reporting!

**Comment by vanhauser-thc** [comment]
fixed with 615ab1a
